api/leaderboard/migrations/0025_alter_unifiedscorehistory_unique_together_and_more.py

```
@@ -0,0 +1,26 @@
+            unique_together=set(),
+        ),
+        migrations.AddField(
+            model_name="unifiedscorehistory",
+            name="user_id",
+            field=models.IntegerField(db_index=True, default=0),
+        ),
+        migrations.AlterUniqueTogether(
+            name="unifiedscorehistory",
+            unique_together={("user_id", "date")},
+        ),
+    ]
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)